File: Java/프로그래머스/0/181885. 할 일 목록/할 일 목록.java
class Solution {
    public String[] solution(String[] todo_list, boolean[] finished) {
        int count=0;
        for(int i=0; i<finished.length ; i++){
            if(finished[i]) {
                continue;
            } 
            else {
                count++;
            }
        }
        String[] answer = new String[count];
        int n=0;
        for(int i=0; i<todo_list.length;i++){
            if(finished[i]) {
                continue;
            }
            else {
                answer[n]=todo_list[i];
                n++;
            }
        }
        return answer;
    }
}

File: 프로그래머스/0/181885. 할 일 목록/할 일 목록.cpp
#include <string>
#include <vector>

using namespace std;

vector<string> solution(vector<string> todo_list, vector<bool> finished) {
    int count=0;
    for(int i=0; i<size(finished) ; i++){
        if(!finished[i]) {
            count++;
        }
    }
    vector<string> answer;
    int n=0;
    for(int i=0; i<size(todo_list);i++){
        if(!finished[i]) {
            answer.push_back(todo_list[i]);
            n++;
        }
    }
    return answer;
}